### PR TITLE
Change node and edge colors

### PR DESCRIPTION
**Pull request by TimSpeelman**
Implements #202 
newdesign1

### CODE REVIEW

**Comment by rkeulemans** [comment]
Can you please add a screenshot of the new design?

File: TriblerGUI/widgets/trustpage/js/request_data.js [hunk 1]
```diff
@@ -49,7 +49,7 @@ function make_cors_request(method, url) {
  * @param callback The callback which is called with the GraphResponseData
  */
 function get_node_info(public_key, neighbor_level, callback) {
-    var url = "http://localhost:8085/multichain/network?focus_node=" + public_key + "&neighbor_level=" + neighbor_level;
+    var url = "http://localhost:8085/multichain/network?focus_node=" + public_key + "&neighbor_level=" + neighbor_level + "&dataset=static";
```
Comment (ghabbenjansen) [inline TriblerGUI/widgets/trustpage/js/request_data.js]: Please revert this change

File: TriblerGUI/widgets/trustpage/js/main.js [hunk 2]
```diff
@@ -206,6 +206,20 @@ function xAtFraction(x0, x1, ratio) {
...
+ * @param nodeB
+ * @returns {number} - the distance in pixels
+ */
+function nodeDistance(nodeA, nodeB){
+    return Math.sqrt(Math.pow(nodeB.x - nodeA.x, 2) + Math.pow(nodeB.y - nodeA.y, 2));
+}
+
+/**
+ * NodeA.x
+ */
```
Comment (ghabbenjansen) [inline TriblerGUI/widgets/trustpage/js/main.js]: Please remove this redundant comment

File: TriblerGUI/widgets/trustpage/js/style_config.js [hunk 3]
```diff
@@ -21,11 +22,12 @@ var config = {
...
             cx : 0,
             cy : 0,
-            cursor : "pointer"
+            cursor : "pointer",
+            strokeWidth: "3px"
         },
         color : {
             domain : [0, 0.5, 1],
-            range : ["red", "yellow", "green"]
+            range : ["#ff1d3e", "#f9ff15", "#0cff18"]
```
Comment (rkeulemans) [inline TriblerGUI/widgets/trustpage/js/style_config.js]: Please use all capital symbols.

File: TriblerGUI/widgets/trustpage/js/style_config.js [hunk 4]
```diff
@@ -3,16 +3,17 @@
  */
 
 var config = {
+    background: "#202020",
     link : {
-        colorLinkSource : "#ffff00",
-        colorLinkTarget : "#ff0000",
+        colorLinkSource : "#22ffd5",
```
Comment (rkeulemans) [inline TriblerGUI/widgets/trustpage/js/style_config.js]: Please use all capital symbols.

File: TriblerGUI/test/widgets/trustpage/js/test_drawing.js [hunk 5]
```diff
@@ -7,6 +7,32 @@ config = require("TriblerGUI/widgets/trustpage/js/style_config.js");
...
+                [1001000000, "1.001 GB"],
+            ];
+
+            data.forEach(function(set){
+                assert.equal(drawing.formatBytes(set[0]), set[1]);
+            });
+
+        });
+
+
```
Comment (ghabbenjansen) [inline TriblerGUI/test/widgets/trustpage/js/test_drawing.js]: Please remove these unnecessary blank lines

File: TriblerGUI/widgets/trustpage/js/main.js [hunk 6]
```diff
@@ -206,6 +206,16 @@ function xAtFraction(x0, x1, ratio) {
 }
 
 /**
+ * Calculate the distance in pixels between nodes
+ * @param nodeA
+ * @param nodeB
+ * @returns {number} - the distance in pixels
+ */
+function nodeDistance(nodeA, nodeB){
```
Comment (rkeulemans) [inline TriblerGUI/widgets/trustpage/js/main.js]: Please add a space here.

File: TriblerGUI/widgets/trustpage/js/drawing.js [hunk 7]
```diff
@@ -17,6 +17,23 @@ function selectNodes(svg) {
...
+function formatBytes(bytes) {
+
+    var sizes = config.byteUnits;
+
+    var i = 0;
+
+    while (bytes >= Math.pow(10, (i+1) * 3) && (i+1) < sizes.length)         i++;
+
+    return parseFloat(Math.round((1.0*bytes) / Math.pow(10, (i-1)*3)) / 1000).toPrecision(4) + " " + sizes[i];
+
```
Comment (ghabbenjansen) [inline TriblerGUI/widgets/trustpage/js/drawing.js]: Again, trailing blank line between final return statement and end of function

File: TriblerGUI/test/widgets/trustpage/js/test_drawing.js [hunk 8]
```diff
@@ -7,6 +7,32 @@ config = require("TriblerGUI/widgets/trustpage/js/style_config.js");
 drawing = require("TriblerGUI/widgets/trustpage/js/drawing.js");
 
 describe("drawing.js", function () {
+    describe('formatBytes', function (){
+        it('should return correct formats', function(){
```
Comment (yinghaodai) [inline TriblerGUI/test/widgets/trustpage/js/test_drawing.js]: Consistency in formatting `function () {`?

File: TriblerGUI/widgets/trustpage/js/style_config.js [hunk 9]
```diff
@@ -3,29 +3,32 @@
  */
 
 var config = {
-    link : {
-        color : "#FFFF00",
-        strokeWidthMin : 6,
-        strokeWidthMax : 10
+    byteUnits: ["B", "KB", "MB", "GB", "TB", "PB"],
```
Comment (rkeulemans) [inline TriblerGUI/widgets/trustpage/js/style_config.js]: `KB` should be `kB` see https://nl.wikipedia.org/wiki/Megabyte

File: TriblerGUI/test/widgets/trustpage/js/test_drawing.js [hunk 10]
```diff
@@ -7,6 +7,32 @@ config = require("TriblerGUI/widgets/trustpage/js/style_config.js");
...
+                [1000000000, "1.000 GB"],
+                [1001000000, "1.001 GB"],
+            ];
+
+            data.forEach(function(set){
+                assert.equal(drawing.formatBytes(set[0]), set[1]);
+            });
+
+        });
+
```
Comment (rkeulemans) [inline TriblerGUI/test/widgets/trustpage/js/test_drawing.js]: Please remove this newline.

File: TriblerGUI/widgets/trustpage/js/drawing.js [hunk 11]
```diff
@@ -17,6 +17,23 @@ function selectNodes(svg) {
...
+}
+
```
Comment (yinghaodai) [inline TriblerGUI/widgets/trustpage/js/drawing.js]: I believe you can the number of new lines for readability purposes

**Comment by yinghaodai** [comment]
retest this please

**Comment by yinghaodai** [comment]
After having been squashed, the commit can be merged!

File: TriblerGUI/widgets/trustpage/js/drawing.js [hunk 12]
```diff
@@ -245,11 +265,20 @@ function drawLinks(svg, data, hoverInfoLabel) {
...
         .duration(1000)
-        .style('opacity', '1');
+        .style("opacity", function (d) {
+            return getLinkOpacity(d);
+        });
 
     return links;
 }
 
+function getLinkOpacity(link){
```
Comment (ghabbenjansen) [inline TriblerGUI/widgets/trustpage/js/drawing.js]: Please add a function description

File: TriblerGUI/widgets/trustpage/js/request_data.js [hunk 13]
```diff
@@ -49,7 +49,7 @@ function make_cors_request(method, url) {
  * @param callback The callback which is called with the GraphResponseData
  */
 function get_node_info(public_key, neighbor_level, callback) {
-    var url = "http://localhost:8085/multichain/network?focus_node=" + public_key + "&neighbor_level=" + neighbor_level;
+    var url = "http://localhost:8085/multichain/network?focus_node=" + public_key + "&neighbor_level=" + neighbor_level +"&dataset=random";
```
Comment (ghabbenjansen) [inline TriblerGUI/widgets/trustpage/js/request_data.js]: Please revert this change

File: TriblerGUI/widgets/trustpage/js/main.js [hunk 14]
```diff
@@ -16,7 +16,7 @@ function onNewData(data) {
 
 // Distance to peers
 var radius_step = config.radius_step,
-    neighbor_level = 2;
+    neighbor_level = 3;
```
Comment (ghabbenjansen) [inline TriblerGUI/widgets/trustpage/js/main.js]: Please revert this change

File: TriblerGUI/widgets/trustpage/js/style_config.js [hunk 15]
```diff
@@ -3,37 +3,43 @@
...
+            range: ["#FF1D3E", "#F9FF15", "#0CFF18"]
         },
         hoverLabel : {
             publicKeyCharacters : 5,
             pageRankDecimals : 4,
             opacity : 0.85
         }
     },
-    radius_step: 120
+    radius_step: 80
```
Comment (ghabbenjansen) [inline TriblerGUI/widgets/trustpage/js/style_config.js]: Maybe make the radius_step dependent on the neighbor level
